terraform.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     azapi = {
       source  = "Azure/azapi"
-      version = ">=2.0"
+      version = ">= 1.12, < 3.0"
```

Comment:
I've tested `1.12` and it cannot work. Could we try `>= 1.13, < 3.0` here?

Reply:
Updated